dask/diagnostics/progress.py

```diff
@@ -37,6 +37,8 @@ class ProgressBar(Callback):
         Width of the bar
     dt : float, optional
         Update resolution in seconds, default is 0.1 seconds
+    out : file object, optional
+        File object to which the progress bar will be written. Default is sys.stdout.
```

Comment:
`sys.stdout` could be enclosed in double backticks `` to use literal markup

Reply:
Perhaps we could also add that the file needs to be opened in text mode (`"w"`) to address the point of confusion.